charts/trento-server/Chart.yaml

```diff
@@ -7,7 +7,7 @@ description: The trento server chart contains all the components necessary to ru
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates
-version: 2.2.1-dev4
+version: 2.2.1-dev5
```

Comment:
Do we need to increment the version every time we add a new API version ? 
Or generally every time we change something in our APIs's ?

Reply:
Everytime something changes in the charts we need to bump this so it works.